electron/src/chatBro/queries/WordCount/WordCount.ts

```diff
@@ -19,28 +20,40 @@ function wordFilter(opts: WordCountTypes.Options): string | undefined {
   if (_.isEmpty(opts.word)) {
     return undefined;
   }
-  return `word = "${opts.word}"`;
+  return `${Columns.WORD} = "${opts.word}"`;
+}
+
+function isEmojiFilter(opts: WordCountTypes.Options): string | undefined {
+  if (opts.isEmoji === true) {
+    return `TRIM(${Columns.WORD}) IN (${emojis})`;
+  }
+  if (opts.isEmoji === false) {
+    return `TRIM(${Columns.WORD}) NOT IN (${emojis})`;
+  }
+  return undefined;
```

Comment:
I prefer readability above all else, but if anyone has any recs on how to make this cleaner, lmk

Reply:
will this return undefined ever get hit? Looks like if first if statement is false, doesn't that guarantee the second one is true

Reply:
You know, I had a shower thought about that ... I was mimicking this off of the other filters we have. 

I will change this.